test/csuite/schema_abort/main.c

```diff
@@ -478,7 +478,7 @@ thread_ts_run(void *arg)
              * availability at older timestamps.
              */
             testutil_check(__wt_snprintf(tscfg, sizeof(tscfg),
-              "oldest_timestamp=%" PRIx64 ",stable_timestamp=%" PRIx64, oldest_ts, oldest_ts));
```

Comment:
Why did you change the order of these? We generally alphabeticalize things where possible, and I think that makes sense here too.

Reply:
OK, that just feel counterintuitive to me. I think we should move stable timestamp first and then oldest timestamp. If the sequence doesn't matter, I'll respect the alphabetic order.